txn_writer.go

```diff
@@ -307,22 +312,25 @@ func (sw *SafeWriter) Write(root string, sm gps.SourceManager, noExamples bool)
 			return errors.Wrap(err, "failed to marshal manifest to TOML")
 		}
 
-		var initOutput string
+		var initOutput []byte
 
 		// If examples are NOT disabled, use the example text
 		if !noExamples {
 			initOutput = exampleTOML
 		}
 
-		// 0666 is before umask; mirrors behavior of os.Create (used by
```

Comment:
Oops, I meant "const", and not actually letting someone change the file mask. Just a suggestion, though.